javascript/javascript1/week4/hw4.js

```diff
@@ -0,0 +1,86 @@
+let todo = [];
+let myName = '';
+const month = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+function getReply(command) {
+
+    if (command.includes('Hello my name is Hamed Hello my name is Hamed')) {
+        return 'Please tell me just once your information';
+
+    } else if (command === 'Hello my name is Hamed') {
+        myName = command.split(' ');
+        return 'nice to meet you ' + myName[4];
+
+    } else if (command === 'Hello my name is') {
+        return 'Please tell your name'
+
+    } else if (command === 'What is my name?') {
+        return myName[4];
+
+    } else if (command === 'add fishing to my todo') {
+
+        todo.push(command.substring(4, 11));
+        return {
+            todo,
+            'firstTodo': 'fishing added to your todo'
+        };
+    } else if (command === 'add singing in the shower to my todo') {
+        todo.push(command.substring(4, 11))
+        return {
+            todo,
+            'secondTodo': 'singing added to your todo'
+        }
+    } else if (command === 'Remove fishing from my todo') {
+        todo.splice(0, 1);
+        return 'Removed fishing from your todo'
```

Comment:
How about splitting the command into an array with `string.split(" ")`, finding the string that needs to be removed as the string between the words "remove" and "from my todo" and then finding the element that needs to be removed ("fishing" - but it could also be anything else) in the todos using `array.indexOf(element)` and removing it using `array.splice()`? 

It may seem more complex but it's a good exercise because you wouldn't know the index of the element you're looking for in advance.

Reply:
Yes It could be. But I do not underestand what is the benefit of using your method as you already mentioned the word you are looking for came after "remove". Actually you know in which position is places the word.